images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -74,7 +76,7 @@ Install-AndroidSDKPackages -AndroidSDKManagerPath $sdkManager `
                           -AndroidPackages $androidToolset.additional_tools
 
 # Android NDK root path.
-$ndkRoot = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
```

Comment:
Do we need to preserve this path for backward compatibility?

Reply:
No, we must not. 
This path containing the spaces was the cause of the whole store and to remove it is the purpose of the PR.

In the issue there's a thread i resisted to make the change and tried to manage out the problem with fixing the google .bat files, but the client has convinced me there's no way but to move the SDK/NDK to the path without the spaces.

The those clients who hardcoded the path will still be able to use it because we have a symbolic link